templates/task-sc.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+    name: skopeo-copy
+    labels:
+        app.kubernetes.io/version: 0.0.1
+spec:
+    description: | 
+        Skopeo is a command line tool for working with remote image registries. 
```

Comment:
This line is too long, usually the description is limited to 80 columns to be nicely displayed later on. Please consider [this example](https://github.com/openshift-pipelines/task-git/blob/fbd3c8b15b10b0deda87538a8cbfb53aada6b1d3/templates/task.yaml#L18-L21).

Reply:
Got it!